packages/gatsby-theme-api-docs/src/components/resource/method/request-response-examples.js

```diff
@@ -58,17 +58,18 @@ const RequestResponseExamples = (props) => {
         </MultiCodeBlock>
       )}
 
-      {responsesCodeExamples.map((codeExample) => {
-        return (
-          <MultiCodeBlock
-            key={`${codeExample.code}-${codeExample.typeDisplayName}`}
-            secondaryTheme={true}
-            title={`${codeExample.code} Response Example: ${codeExample.typeDisplayName}`}
-          >
-            <CodeBlock language="json" content={codeExample.value} />
-          </MultiCodeBlock>
-        );
-      })}
+      {Boolean(responsesCodeExamples.length) &&
```

Comment:
why do we need this additional check? Isn't `responsesCodeExamples` an array? If the array is empty, map function doesn't get executed